__tests__/app.test.js

```diff
@@ -77,3 +80,47 @@ describe("2. GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("3. GET /api/users", () => {
+  test("200: respond with an array of user objects containing the correct keys and properties", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
```

Comment:
when working wtih arrays it would be nice to check the length to make sure we're getting everything back we need